### PR TITLE
Added content to package.json

### PR DESCRIPTION
**Pull request by baderbuddy**
Added content to the test package.json so it's not an empty file.

### CODE REVIEW

File: test/fixtures/config/package.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{)
```
Comment (KenanY) [inline test/fixtures/config/package.json]: Perhaps this should at least be `{}`?

Comment (baderbuddy) [inline test/fixtures/config/package.json]: Wow, yeah I'll fix the typo.

**Review by Andreslyn — APPROVED** [review]

**Review by Andreslyn — APPROVED** [review]

**Review by Andreslyn — APPROVED** [review]

**Review by Andreslyn — APPROVED** [review]

**Comment by iarna** [comment]
@baderbuddy It looks like you can actually just remove the file entirely. I'd prefer that, I think.

**Review by iarna — APPROVED** [review]

**Comment by iarna** [comment]
Thanks for this! I've squashed the commits and rewritten the commit message to match our norms and it's now merged into `release-next` and will be a part of the release this Thursday (2016-11-17).

**Pull request closed without merging**